tabs/tab_baseline_matching.py

```
@@ -859,14 +902,52 @@ def ui_balance_alert():
                 )
             )
 
-    @render_widget
+    @render.ui
     def out_love_plot():
```

Ruff (0.14.10)

Missing return type annotation for private function `out_love_plot`

(ANN202)